src/components/RNMarkdownTextInput.tsx

```diff
@@ -13,6 +13,17 @@ type AnimatedMarkdownTextInputRef = typeof AnimatedMarkdownTextInput & MarkdownT
 
 type RNMarkdownTextInputProps = Omit<MarkdownTextInputProps, 'parser'>;
 
+function handleFormatSelection(selectedText: string, formatCommand: string) {
+    switch (formatCommand) {
+        case 'formatBold':
+            return `*${selectedText}*`;
```

Comment:
Is it possible to move this to a util or use any method from ExpensifyCommon? Because adding `*` and `_` wouldn't cover all cases.

Reply:
I don't think it's necessary to move it to a util since it's only being used here.

> Because adding * and _ wouldn't cover all cases.

I don't quite get this one.

Reply:
I strongly feel it should be in the utils. Consider this we'll be adding the same behavior for keyboard shortcuts. I'll let @blimpich decide on this.

> > Because adding * and _ wouldn't cover all cases.
>
> I don't quite get this one.

It's about clicking the button several times discussed [here](https://github.com/Expensify/App/pull/54983#issuecomment-2581167286). We can ignore it in this comment.

Reply:
If we're adding the keyboard shortcuts, it'll probably be added to the live markdown repo too which we will handle on this component too.

Reply:
I don't feel strongly about this right now. Lets make it a utility in a follow up. I'll cc you both in the follow up issue.